specification/web/resource-manager/Microsoft.Web/stable/2020-09-01/WebApps.json

```diff
@@ -1504,6 +1546,245 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/config/configreferences/appsettings": {
```

Comment:
Nested resource appsettings should be in lowerCamelCase -> appSettings

Reply:
Keeping consistent with existing API.